flink-core/src/main/java/org/apache/flink/configuration/HighAvailabilityOptions.java

```diff
@@ -204,6 +204,23 @@
                     .withDescription(
                             "The time before a JobManager after a fail over recovers the current jobs.");
 
+    /**
+     * Safety hatch to fallback to the old ha services implementations.
+     *
+     * <p>Ideally, we can remove this option together with the old implementations in the next
+     * release.
+     */
+    @Documentation.Section(Documentation.Sections.EXPERT_HIGH_AVAILABILITY)
+    public static final ConfigOption<Boolean> USE_OLD_HA_SERVICES =
+            key("high-availability.use-old-ha-services")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "Use this option to disable the new HA service implementations for ZooKeeper and K8s. This is safety hatch in case that the new ha services are buggy.")
```

Comment:
nit: "[...] This is a safety hatch [...]"

Reply:
Will correct it.